jormungandr/Cargo.toml

```diff
@@ -52,7 +52,7 @@ slog-scope = "4.1"
 slog-term = "2.4.0"
 structopt = "^0.3"
 thiserror = "1.0.16"
-tokio = { version = "0.2.12", features = ["signal"] }
+tokio = { version = "0.2.12", features = ["rt-threaded", "time", "sync", "stream", "signal", "test-util", "blocking"] }
```

Comment:
So many features are added with this small change, are these needed for other things to work?

Reply:
I guess it's leaner than "full" in any case...